apps/examples/webserver_test/webserver_test_main.c

```diff
@@ -0,0 +1,1299 @@
+	printf("   $ webserver_test <iterations> <method> <uri> [options...] \n");
+	printf("\n");
+	printf(" <method>   : %%s (GET, PUT, POST, DELETE)\n");
+	printf(" <uri>      : %%s (Host address : should be started with http:// or https://)\n");
+	printf("\n [options...] \n");
+	printf(" test_entity=SIZE      Test entity size (default is 0)\n");
+	printf(" chunk-count=Y    	   Turns on Chunked-Transfer and sends Y chunks of test_entity (default Chunked-Transfer is OFF)\n");
+	printf(" keep-alive=X          Turns on keep-alive and sends X requests (default Keep-Alive is OFF)\n");
+	printf("\n  example:\n");
+	printf("   $ webserver_test 3 GET https://127.0.0.1/\n");
+	printf("   $ webserver_test 3 GET https://127.0.0.1/ test_entity=5\n");
+}
+
+static char* ws_test_wget_strcpy(char *dest, const char *src, struct http_client_request_t *ws)
+{
+	int len = strlen(src);
+
+	if (dest + len - ws->buffer >= ws->buflen) {
+		PRNT("Error: buffer is too small");
+		return NULL;
+	}
+
+	memcpy(dest, src, len);
+	dest[len] = '\0';
+	return dest + len;
+}
+
+static char* ws_test_wget_strlencpy(char *dest, const char *src, int len, struct http_client_request_t *ws)
+{
+	if (dest + len - ws->buffer >= ws->buflen) {
+		PRNT("Error: buffer is too small");
+		return NULL;
+	}
+
+	memcpy(dest, src, len);
+	dest[len] = '\0';
+	return dest + len;
+}
+
+static char* ws_test_wget_chunksize(char *dest, int len, struct http_client_request_t *ws)
+{
+	FUNC_EN;
+	char size[10] = {0,};
+
+	if (!dest) {
+		return NULL;
+	}
+
+	snprintf(size, sizeof(size), "%x", len);
+	len = strlen(size);
+	if (dest + len - ws->buffer >= ws->buflen) {
+		PRNT("Error: buffer is too small");
+		return NULL;
+	}
+	memcpy(dest, size, len);
+	FUNC_EX;
+	return dest + len;
+}
+
+
+
+int ws_test_http_client_response_init(struct http_client_response_t *response)
+{
+	FUNC_EN;
+	if(response == NULL){
+		PRNT("Error: NULL Response");
+		return -1;
+	}
+	response->phrase = malloc(WEBCLIENT_CONF_MAX_PHRASE_SIZE);
+	if (response->phrase == NULL) {
+		PRNT("Error: Fail to init");
+		return -1;
+	}
+	response->message = malloc(WEBCLIENT_CONF_MAX_MESSAGE_SIZE);
+	if (response->message == NULL) {
+		PRNT("Error: Fail to init");
+		free(response->phrase);
+		return -1;
+	}
+	response->headers = malloc(sizeof(struct http_keyvalue_list_t));
+	if (response->headers == NULL || http_keyvalue_list_init(response->headers) < 0) {
+		PRNT("Error: Fail to init");
+		free(response->phrase);
+		free(response->message);
+		return -1;
+	}
+	FUNC_EX;
+	return 0;
+}
+
+void ws_test_http_client_response_release(struct http_client_response_t *response)
+{
+	FUNC_EN;
+	free(response->phrase);
+	free(response->message);
+	if (http_keyvalue_list_release(response->headers) != 0) {
+		PRNT("http_keyvalue_list_release(response->headers) failed");
+	}
+	free(response->headers);
+}
+
+static int ws_test_wget_gethostip(FAR char *hostname, in_addr_t *ipv4addr)
+{
+	FUNC_EN;
+#ifdef CONFIG_NET_LWIP_NETDB
```

Comment:
Add Kconfig dependency if CONFIG_NET_LWIP_NETDB is mandatory. 
Following code is example.
```
config NET_ETH_MTU
	int "Ethernet packet buffer size (MTU)"
	default 1294 if NET_IPv6
	default 590 if !NET_IPv6
	depends on NET_ETHERNET
	range 590 1518
```

Reply:
Made required changes